ios/Samples/AndorraTelecomPublic/Podfile

```diff
@@ -11,6 +12,9 @@ end
 post_install do |installer|
 	installer.pods_project.targets.each do |target|
 		target.build_configurations.each do |config|
+			if target.to_s == 'Hokusai'
+				config.build_settings['SWIFT_VERSION'] = '3.2'
```

Comment:
It seems that `Hokusai` library provides support to Swift 4, isn't it?

Reply:
Not in the momment I tried that. I think the conversion is badly done